src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx

```diff
@@ -87,7 +92,49 @@ export default function StoryCard({ story }: Props) {
           <hr className="my-16 bg-gray-200" />
           <div className="flex gap-24 items-center">
             <VoteButton votes={story.votes_count} dense onVote={vote} />
+
+            <span className="text-gray-400">
+              <FiMessageSquare />{" "}
+              <span className="align-middle">
+                {isLoadingComments ? (
+                  <Skeleton width="9ch" />
+                ) : (
+                  `${comments.data.length} Comments`
+                )}
+              </span>
+            </span>
           </div>
+          {comments?.data && comments.data.length > 0 && (
+            <div className=" bg-gray-100 p-24 mt-16 rounded">
+              <ul className="flex flex-col gap-16">
+                {comments?.data.slice(0, 3).map((comment) => (
+                  <li>
+                    <p>
+                      <span className="font-bold text-gray-900">
+                        {nip19.npubEncode(comment.pubkey).slice(0, 10)}
+                      </span>{" "}
+                      <span className=" text-body5">
+                        {calcTimeSincePosting(
+                          new Date(comment.created_at * 1000).toISOString()
+                        )}
+                      </span>
+                    </p>
+                    <p className="text-gray-600 line-clamp-2">
```

Comment:
```suggestion
                    <p className="text-gray-600">
```

lets just show the whole comment.

Reply:
![image](https://user-images.githubusercontent.com/50504183/230607521-8bb5c284-ab89-4cb0-a81d-eebd91d42a54.png)

Some comments could be very long...
See example above, and this is only 1 comment.